src/sentry/static/sentry/app/views/organizationIntegrations/sentryAppInstallations.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {groupBy} from 'lodash';
+import parseurl from 'parseurl';
+import qs from 'query-string';
+import {addErrorMessage, addSuccessMessage} from 'app/actionCreators/indicator';
+import AsyncView from 'app/views/asyncView';
+import {Client} from 'app/api';
+import SentryApplicationRow from 'app/views/settings/organizationDeveloperSettings/sentryApplicationRow';
+import {t} from 'app/locale';
+
+const api = new Client();
```

Comment:
We should use the higher order component: `withApi` instead. It's basically a replacement for `ApiMixin` since we want to move to ES6 class components. Besides providing the api client in props as `props.api`, it aborts ongoing requests if the component is unmounted.